src/status_im/contexts/wallet/common/utils/send.cljs

```diff
@@ -3,20 +3,16 @@
 
 (defn calculate-gas-fee
   [data]
-  (let [gas-amount       (money/bignumber (get data :gas-amount))
-        gas-fees         (get data :gas-fees)
-        eip1559-enabled? (get gas-fees :eip1559-enabled)
-        billion          (money/bignumber "1000000000")]
-    (if eip1559-enabled?
-      (let [base-fee      (money/bignumber (get gas-fees :base-fee))
-            priority-fee  (money/bignumber (get gas-fees :max-priority-fee-per-gas))
-            fee-with-tip  (money/bignumber (money/add base-fee priority-fee))
-            total-gas-fee (money/mul gas-amount fee-with-tip)]
-        (money/with-precision (money/div total-gas-fee billion) 10))
-      (let [gas-price     (money/bignumber (get gas-fees :gas-price))
-            total-gas-fee (money/mul gas-amount gas-price)]
-        (money/with-precision (money/div total-gas-fee billion) 10)))))
+  (let [gas-amount         (money/bignumber (get data :gas-amount))
+        gas-fees           (get data :gas-fees)
+        eip1559-enabled?   (get gas-fees :eip-1559-enabled)
+        optimal-price-gwei (money/bignumber (if eip1559-enabled?
+                                              (get gas-fees :max-fee-per-gas-medium)
+                                              (get gas-fees :gas-price)))
+        total-gas-fee-wei  (money/mul (money/->wei :gwei optimal-price-gwei) gas-amount)
+        l1-fee-wei         (money/->wei :gwei (get gas-fees :l-1-gas-fee))]
```

Comment:
this should always be 0 unless it is an Optimism (or Optimism Sepolia) transaction